variables.tf

```diff
@@ -78,12 +78,24 @@ variable "tags" {
   default     = {}
 }
 
+variable "policy_arn" {
```

Comment:
Please change this to a list and rename to `policy_arns` so it supports multiple policy ARN attachments.

Reply:
Introduced a list of policies.